Rakefile

```diff
@@ -1,6 +1,28 @@
 require "yast/rake"
 
+AUTOINST_DIR = Packaging::Configuration::DESTDIR + "/usr/share/autoinstall/"
+
 Yast::Tasks.configuration do |conf|
   #lets ignore license check for now
   conf.skip_license_check << /.*/
+  conf.install_locations["scripts/*.service"] =
+    Packaging::Configuration::DESTDIR + "/usr/lib/systemd/system/"
+  conf.install_locations["xslt/*.xslt"] = AUTOINST_DIR + "/xslt/"
```

Comment:
For file paths I prefer `File.join`, that works better and is error proof, e.g. `dir + "subdir"` breaks when dir is e.g. `/usr/share`. You need to be careful when using +.

Reply:
OK, lets keep it as it is.